onnx/defs/reduction/defs.cc

```diff
@@ -221,7 +221,8 @@ If keepdims equal 0, then the resulting tensor have the reduced dimension pruned
 If select_last_index is True (default False), the index of the last occurrence of the {name} 
 is selected if the {name} appears more than once in the input. Otherwise the index of the 
 first occurrence is selected.
-The type of the output tensor is integer.)DOC";
+The type of the output tensor is integer.
```

Comment:
This the wrong method. You are adding this to ArgReduceDocGenerator. It should be added to ReduceDocGenerator.
Also all reduction ops use ReduceDocGenerator so you need to add a conditional whether complementary axis should be added or not

Reply:
Updated. Thanks